### ISSUE TITLE
Send runtimedata into unleash.

### ISSUE THREAD

**Issue by sveinelo** [Unleash/unleash-client-java#22]
``` java
public interface Unleash {
...
    boolean isEnabled(String toggleName, Map<String,String> parameters);
...
}
```

**Comment by ivarconr** [comment]
Yes, this is how it is solved in the node-client and should probably be added to the java-client also. 

The only issue is that it is a bit cumbersome to do inline maps in java:

```java
unleash.isEnabled("someToggle", new HashMap<String, String>(){{
       put("userId","123");}};));
```

**Comment by stigkj** [comment]
It would be possible to use varargs.

**Comment by ivarconr** [comment]
will be solved as part of #33
